x-pack/plugins/upgrade_assistant/server/lib/__fixtures__/fake_deprecations.json

```diff
@@ -19,6 +19,12 @@
       "message": "Datafeed [deprecation-datafeed] uses deprecated query options",
       "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking-changes-7.0.html#breaking_70_search_changes",
       "details": "[Deprecated field [use_dis_max] used, replaced by [Set [tie_breaker] to 1 instead]]"
+    },
```

Comment:
The mock JSON has been updated to include additional cases that weren't previously covered in the tests: old ML model snapshots, deprecated index settings, and closed indices.